index.html

```diff
@@ -14,6 +16,95 @@
+            <a href="services.html" id="nav-contact">Contact</a>
+          </nav>
+
+          <div>
+          <img id="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </div>
+        </header>
+      
+        <div id="header-row">
+          <div class="demand">
+            <h1>
+              Innovation<br>
+              On<br>
+              Demand
+            </h1>
+            <a href="#"><div class="get-started">Get Started</div></a>
+          </div>
+          
+          <div class="circle-img">
+            <img src="img/header-img.png" alt="Image of a code snippet.">
+          </div>
+        </div>
+        
+        <main>
+          <section class="top-section">
+            <div class="features">
+              <h3>Features</h3>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+            </div>
+            
+            <div class="about">
+              <h3>About</h3>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+            </div>
+          </section>
+    
+          <img class="middle-img" src="img/mid-page-accent.jpg"  alt="Image of code snippets across the screen">
+          
+          <section class="bottom-section">
+            <div class="services">
+              <h3>Services</h3>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+            </div>
+            
+            <div class="product">
+              <h3>Product</h3>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+            </div>
+            
+            <div class="vision">
+              <h3>Vision</h3>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+            </div>
+          </section>
+        </main>
+      
+        <footer>
+          <h3>Contact</h3>
+          <address>
+            123 Way 456 Street<br>
+            Somewhere, USA<br><br>
+            1 (888) 888-8888<br><br>
+            
+            <a href="mailto:sales@greatidea.io">sales@greatidea.io</a>
+          </address>
+    
+          <copyright>Copyright Great Idea! 2018</copyright>
+        </footer>
+    
+      </div>
 </body>
```

Comment:
Your HTML is really clean and easy to read I would keep consistent with the css too